mscore/timeline.cpp

```diff
@@ -2882,11 +2885,10 @@ const TimelineTheme& Timeline::activeTheme() const
 
 void Timeline::updateTimelineTheme()
       {
-      scene()->setBackgroundBrush(QBrush(activeTheme().backgroundColor));
-      updateView();
+      QBrush backgroundBrush = QBrush(activeTheme().backgroundColor);
```

Comment:
Would you mind making this `const QBrush`?

Reply:
Done.